exasol/slc/tool/commands/deploy.py

```diff
@@ -157,5 +157,23 @@ def deploy(
             ssl_cert_path=ssl_cert_path,
             use_ssl_cert_validation=use_ssl_cert_validation,
         )
-        with result.open("r") as f:
-            print(f.read())
+        for flavor_name, lang_def_builds_per_release in result.items():
+            for release, deploy_result in lang_def_builds_per_release.items():
+                print(
```

Comment:
We probably should make the print configurable, depending on circumstances the function is used

Reply:
Hm, I mean in most cases probably the API method will be used. I wouldn't spend too much effort on this